Travailler ensemble sans subordination

DavidBruant opened the pull request.

DavidBruant committed:
premier brouillon Travailler ensemble sans subordination

davidbgk commented on travailler-ensemble-sans-subordination.md:
```diff
@@ -0,0 +1,137 @@
+
+Le lien de subordination n'est pas défini dans le Code du Travail
+
+Il est défini par un [par la jurisprudence](https://www.legifrance.gouv.fr/juri/id/JURITEXT000007035180/)
```
```suggestion
Il est défini [par la jurisprudence](https://www.legifrance.gouv.fr/juri/id/JURITEXT000007035180/)
```

yaf committed:
revois une formulation et une coquille

Et peut-être que c'est pas vraiment des erreurs mais moi qui n'avais pas bien compris la tournure de la phrase ?

DavidBruant committed:
Update travailler-ensemble-sans-subordination.md

Co-authored-by: David Larlet <3556+davidbgk@users.noreply.github.com>

DavidBruant committed:
Rename travailler-ensemble-sans-subordination.md to _posts/2020/11/20/travailler-ensemble-sans-subordination.md
